prov/efa/src/efa_cntr.c

```diff
@@ -178,6 +178,24 @@ static void efa_rdm_cntr_progress(struct util_cntr *cntr)
 	ofi_genlock_unlock(&cntr->ep_list_lock);
 }
 
+static void efa_cntr_progress(struct util_cntr *cntr)
+{
+	struct util_ep *ep;
+	struct fid_list_entry *fid_entry;
+	struct dlist_entry *item;
+
+	ofi_genlock_lock(&cntr->ep_list_lock);
+	dlist_foreach(&cntr->ep_list, item) {
```

Comment:
We should avoid iterating `ep_list`, but iterating the ibv cq list as the rdm code path does. This is a pain for cntr as it doesn't have a 1-to-1 relationship with ibv cq.

Reply:
But wait, you cannot bind 1 cntr to multiple eps, so that ep_list is just a 1 element array. Then we don't need to bother changing the logic here.

Reply:
Ignore the last comment, you can bind 1 cntr to multiple eps and what you read will be the aggregation of completions from multiple eps.